CUDA 11 Test: TestArrayElementwiseOp::test_doubly_broadcasted_pow

## Failing Tests

```
tests/cupy_tests/core_tests/test_ndarray_elementwise_op.py::TestArrayElementwiseOp::test_doubly_broadcasted_pow FAILED                                                                                 [100%]
```

It seems there is a precision decrease in CUDA 11.
This happens only when one of the operand given to `pow` is `complex64` dtype.

```
E       AssertionError: 
E       Not equal to tolerance rtol=1e-07, atol=0
E       
E       Mismatched elements: 1 / 18 (5.56%)
E       Max absolute difference: 1.5258789e-05
E       Max relative difference: 1.2207032e-07
E        x: array([[[  1.      +0.j,   2.      +0.j,   3.      +0.j],
E               [  1.      +0.j,   4.      +0.j,   9.      +0.j],
E               [  1.      +0.j,   8.      +0.j,  27.000002+0.j]],...
E        y: array([[[  1.+0.j,   2.+0.j,   3.+0.j],
E               [  1.+0.j,   4.+0.j,   9.+0.j],
E               [  1.+0.j,   8.+0.j,  27.+0.j]],...
```

### Enviornment

```
cu>>> cupy.show_config()
CuPy Version          : 8.0.0b5
CUDA Root             : /usr/local/cuda
CUDA Build Version    : 11000
CUDA Driver Version   : 11000
CUDA Runtime Version  : 11000
cuBLAS Version        : 11100
cuFFT Version         : 10200
cuRAND Version        : 10201
cuSOLVER Version      : (10, 5, 0)
cuSPARSE Version      : 11100
NVRTC Version         : (11, 0)
Thrust Version        : 100909
CUB Build Version     : 100909
cuDNN Build Version   : None
cuDNN Version         : None
NCCL Build Version    : None
NCCL Runtime Version  : None
cuTENSOR Version      : None
```

--- a/tests/cupy_tests/core_tests/test_ndarray_elementwise_op.py
+++ b/tests/cupy_tests/core_tests/test_ndarray_elementwise_op.py
@@ -315,7 +315,7 @@ class TestArrayElementwiseOp(unittest.TestCase):
         self.check_array_broadcasted_op(operator.ne)
 
     @testing.for_all_dtypes_combination(names=['x_type', 'y_type'])
-    @testing.numpy_cupy_allclose()
+    @testing.numpy_cupy_allclose(rtol=1e-6)
     def check_array_doubly_broadcasted_op(self, op, xp, x_type, y_type,
                                           no_bool=False, no_complex=False):
         x_dtype = numpy.dtype(x_type)

